configure.ac

```diff
@@ -76,6 +76,12 @@ AC_ARG_ENABLE(ssh, AS_HELP_STRING([--disable-ssh], [Disable cockpit-ssh build an
 AM_CONDITIONAL(WITH_COCKPIT_SSH, test "$enable_ssh" != "no")
 AC_MSG_RESULT(${enable_ssh:=yes})
 
+# --disable-bridge
```

Comment:
I find this rather confusing -- this doesn't disable the bridge at all, there will still be a /usr/bin/cockpit-bridge . What happened to `--enable-pybridge` or `-python`?

Reply:
From our discussion: I'll switch this back to `--enable-python`, move the python installation (and moving askpass to libexecdir) back into `make install`, switch to a PEP517 install, and remove `setup.py`.